src/common/Touchable.js

```diff
@@ -53,7 +54,7 @@ export default class Touchable extends PureComponent<Props> {
           onPress={onPress}
           onLongPress={onLongPress}
         >
-          <View>{children}</View>
+          <View style={wrapperStyle}>{children}</View>
```

Comment:
`TouchableHighlight` is different from `TouchableNativeFeedback`. 
`TouchableHighlight` don't have `flex:1` by default on iOS (or may on Android too). But `TouchableNativeFeedback` have, at least on Android. So if we directly apply `style` to `view` then, then it will break in components like `Buttons` as then touchable will only be region on text not full button. 

And if we always provide `flex: 1` to `TouchableHighlight`, then it will break where there are 2-3 touchable in single view, as then all touchable will take equal space. 

So we can't say that we want `flex:1` always on `TouchableHighlight`, but sometimes we want. Thus this need to be configurable.

And styles like `flexDirection` needs to be provided to `view`, because `TouchableHighlight` has only one child so   `flexDirection` has no effect. 

So it's better to have `style` & `wrapperStyle`.